src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -212,6 +212,12 @@
                        TextWrapping="Wrap"
                        Margin="{StaticResource SmallTopMargin}"/>
 
+            <TextBlock x:Uid="Run_PluginsLoading"
+                       Foreground="{ThemeResource SystemControlErrorTextForegroundBrush}"
```

Comment:
Why error brush. It's no error.

Reply:
We use that brush for warnings

Reply:
I guess we should use info style here

Reply:
For me this is an info and not an error or warning.

Reply:
@enricogior
But we should change color before merging. I pointed the out the reason above.

Reply:
Changed it to regular text